### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by alexdesiqueira**
Improving info on README and updating example images.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,26 +15,42 @@ Run the `pipeline.py` file with the arguments to read in raw images and output r
...
 * `-dpi` : Optional argument to specify resolution of the output image. (Default is `300`.)
 * `-g`, `--grabcut` : Use OpenCV's grabcut method in order to improve binarization on blue butterflies.
 * `-u`, `--unet` : Use the U-net deep neural network in the binarization step.
 
+## Measurement results
+
+The file with measurement results (default: `results.csv`), returned by this pipeline, contains the following measurements:
+
+* `image_name`, the filename of the processed image.
+* `dist_mm["dist_l"]`, distance from the tip of the left wing to the left shoulder.
```
Comment (stefanv) [inline README.md:42]: Is this the correct header name?

**Review by stefanv — COMMENTED** [review]
Thanks, @alexdesiqueira.  Go ahead and merge once you've ensured that the header names in results.csv match what's described here.

File: README.md (hunk 1 above)
Comment (alexdesiqueira) [reply README.md]: You're right @stefanv, I got the names of the variables instead :sweat_smile: Fixed.

**Comment by stefanv** [comment]
Nice, thanks!

**Pull request merged**